src/loader.ts

```diff
@@ -21,11 +27,19 @@ export async function loadAll<T>(dir: string, schema?: Schema): Promise<T[]> {
 }
 
 export async function load<T>(filePath: string, schema?: Schema): Promise<T> {
-  const isValid = schema ? ajv.compile(schema) : () => true;
   const text = await fs.readFile(filePath, 'utf8');
   const item = yaml.load(text) as T;
-  if (!isValid(item)) {
-    throw new Error(`Invalid entity: ${JSON.stringify(item)}`);
+  if (!schema) {
+    return item;
+  }
+
+  const isValid = ajv.compile<T>(schema);
+  if (
+    !isValid(item) &&
+    !(isValid.errors === null || isValid.errors === undefined)
+  ) {
```

Comment:
```suggestion
  if (!isValid(item)) {
```

Per [Validation errors](https://ajv.js.org/api.html#validation-errors), a validation failure should populate the `errors` property of the validation function - so it's safe to simplify this a bit

Reply:
Thanks for the suggestion, it's been committed.